iquip/apps/logger.py

```diff
@@ -1,11 +1,14 @@
 """App module for log viewer in apps."""
 
 import logging
+from logging import handlers
 from typing import Any, Optional, Tuple, Callable
+from functools import partial
```

Comment:
In most cases, importing a function instead of its module should be avoided:
```python
import functools

functools.partial(...)
```
Although sometimes we use it for some popular functions for convenience.
Of course `partial()` is popular enough, but let's keep our convention.